thirdparty/s3.go

```diff
@@ -10,18 +10,23 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 )
 
-//PresignExpireTime sets the amount of time the link is live before expiring
-const PresignExpireTime = 24 * time.Hour
+const (
+	// PresignExpireTime sets the amount of time the link is live before expiring.
+	PresignExpireTime = 24 * time.Hour
+	// NotFound is returned by S3 when an object does not exist.
+	NotFoundError = "NotFound"
```

Comment:
Can we either call this S3NotFoundError (since thirdparty has more than s3 in it) or define it as a local constant in s3_put

Reply:
I moved it to s3_put